task2-bash/backup.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash -ex
 
 src_folder="../task1-simple-program"
-tmp_folder="tmp/guesanumber"
+tmp_folder="/tmp/guesanumber"
 
 
+## Flushing tmp directory
+[ -d $tmp_folder ] && rm $tmp_folder/*
```

Comment:
rm -r may neccessary

Reply:
I didn't use -r key here intentionally to affect only files